index.js

```diff
@@ -250,7 +274,10 @@ function decode(input) {
     else if (t.equals(MSG_T)) return decoder.message(input)
     else if (t.equals(BLOB_T)) return decoder.blob(input)
     else if (t.equals(BOX_T)) return decoder.box(input)
-    else if (tf.equals(SIGNATURE_TF)) return decoder.signature(input)
+    else if (t.equals(SIGNATURE_T)) {
```

Comment:
I modified this so that we can catch decoding of buffers which are `type: signature`, but have an unknown `format`, and throw.

Otherwise these were passed through as un-decoded buffers
:fire: (breaking change)